api/registrations/serializers.py

```diff
@@ -278,13 +279,22 @@ def create(self, validated_data):
         registration_choice = validated_data.pop('registration_choice', 'immediate')
         embargo_lifted = validated_data.pop('lift_embargo', None)
         reviewer = is_prereg_admin_not_project_admin(self.context['request'], draft)
+        children = validated_data.pop('children', None)
```

Comment:
`RegistrationSerializer` works as kind of a de facto write-only serializer, because children is also a RelationshipField on `BaseRegistrationSerializer` it causes many tests to fail.